californium-core/src/test/java/org/eclipse/californium/core/test/Issue834Test.java

```diff
@@ -0,0 +1,281 @@
+package org.eclipse.californium.core.test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import org.eclipse.californium.core.CoapClient;
+import org.eclipse.californium.core.CoapResource;
+import org.eclipse.californium.core.CoapResponse;
+import org.eclipse.californium.core.CoapServer;
+import org.eclipse.californium.core.coap.CoAP.Code;
+import org.eclipse.californium.core.coap.CoAP.ResponseCode;
+import org.eclipse.californium.core.coap.Request;
+import org.eclipse.californium.core.network.config.NetworkConfig;
+import org.eclipse.californium.core.server.resources.CoapExchange;
+import org.eclipse.californium.rule.CoapNetworkRule;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+/**
+ * Test to clarify Californium issue #834
+ * 
+ * @author Rogier Cobben
+ * 
+ *         Results : Failed tests:
+ *         testGetWithLargePayload(nl.teslanet.test.cf.Issue834Test): wrong
+ *         content length returned: expected:<8192> but was:<2012>
+ *         testDeleteWithLargePayload(nl.teslanet.test.cf.Issue834Test): wrong
+ *         content length returned: expected:<8192> but was:<2012>
+ *         testPatchWithLargePayload(nl.teslanet.test.cf.Issue834Test): wrong
+ *         content length returned: expected:<8192> but was:<2012>
+ *         testFetchWithLargePayload(nl.teslanet.test.cf.Issue834Test): wrong
+ *         content length returned: expected:<8192> but was:<2012> Tests run:
+ *         12, Failures: 4, Errors: 0, Skipped: 0
+ */
```

Comment:
Is it intended to add this tests class in this PR ?

Reply:
Uups, I asked for that to be contributed by the author. It's "on the way", I remove it here.

Reply:
It's now PR #849